databuilder/databuilder/models/table_stats.py

```diff
@@ -10,20 +10,128 @@
 from databuilder.models.graph_node import GraphNode
 from databuilder.models.graph_relationship import GraphRelationship
 from databuilder.models.graph_serializable import GraphSerializable
-from databuilder.models.table_metadata import ColumnMetadata
+from databuilder.models.table_metadata import ColumnMetadata, TableMetadata
 from databuilder.models.table_serializable import TableSerializable
 
+LABEL = 'Stat'
+STAT_RESOURCE_RELATION_TYPE = 'STAT_OF'
+RESOURCE_STAT_RELATION_TYPE = 'STAT'
+
+
+class TableStats(GraphSerializable, TableSerializable):
+    """
+    Table stats model.
+    """
+
+    KEY_FORMAT = '{db}://{cluster}.{schema}' \
+                 '/{table}/{stat_type}/'
+
+    def __init__(self,
+                 table_name: str,
+                 stat_name: str,
+                 stat_val: str,
+                 is_metric: bool,
+                 db: str = 'hive',
+                 schema: str = None,
+                 cluster: str = 'gold',
+                 start_epoch: str = None,
+                 end_epoch: str = None
+                 ) -> None:
+        if schema is None:
+            self.schema, self.table = table_name.split('.')
+        else:
+            self.table = table_name
+            self.schema = schema
+        self.db = db
+        self.start_epoch = start_epoch
+        self.end_epoch = end_epoch
+        self.cluster = cluster
+        self.stat_type = stat_name
```

Comment:
consistent naming plz

Reply:
probably by renaming the instance variable, not the argument name. Since the existing model uses that arg name too and that's more likely to cause issues.